build.sh

```diff
@@ -15,7 +15,7 @@ export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 export DOTNET_MULTILEVEL_LOOKUP=0
 export NUGET_PACKAGES="$SCRIPT_ROOT/packages/"
 
-"$SCRIPT_ROOT/init-tools.sh"
+. "$SCRIPT_ROOT/init-tools.sh"
```

Comment:
Does it work to use the actual `source "$DIR/common/_prettyprint.sh"` command?  That seems more obvious to me when reading the code than trying to see that little `.`.

Reply:
`source` isn't part of POSIX, so `.` is in theory more portable.

Reply:
We have #!/bin/bash at the top...

Reply:
Hmm, more real than I thought: `dash` apparently doesn't support `source`. https://stackoverflow.com/a/13702876 https://wiki.ubuntu.com/DashAsBinSh#source

Reply:
👍

Reply:
(Whoops, didn't refresh and missed your comment.)

You're right that we do have that direct dependency on `bash` right now, and it would be ok to use `source`. A while back someone pointed out some bashisms in BuildTools and I think CoreFX. I've pointed out obvious ones like `[[` since then, but I don't know if there's a higher-level drive to avoid a `bash` dependency.

Reply:
Right now we are definitely dependent on bash.

Reply:
I'll go ahead and change it to `source` then--the readability concern also came up in the CoreFX PR. It's not like this would be hard to spot/fix if we do want to work on non-`bash`.